docs/metadata.md

```diff
@@ -0,0 +1,582 @@
+# Cachito request metadata
```

Comment:
Wdyt about adding an index for all lvl 2 titles (##) right under the main title?

Reply:
I like it :+1:

Reply:
Added